assembly/pom.xml

```diff
@@ -666,6 +666,602 @@
+                                            <username>mapr</username>
+                                            <groupname>mapr</groupname>
+                                            <sources>
+                                                <source>
+                                                    <location>${envClassifier}/src/main/resources/examples/conf/hbase-env.sh</location>
+                                                </source>
+                                                <source>
+                                                    <location>${envClassifier}/src/main/resources/examples/conf/hbase-site.conf.xml</location>
+                                                    <destination>hbase-site.xml</destination>
+                                                </source>
+                                            </sources>
+                                            <filemode>755</filemode>
+                                        </mapping>
+                                        <mapping>
+                                            <directory>/opt/mapr/hbase/${envHbase}/conf</directory>
+                                            <configuration>true</configuration>
+                                            <username>mapr</username>
+                                            <groupname>mapr</groupname>
+                                            <filemode>755</filemode>
+                                        </mapping>
+                                    </mappings>
+                                    <cleanScriptlet>
+                                        <script>rm -r %{buildroot}</script>
+                                    </cleanScriptlet>
+                                    <installScriptlet>
+                                        <script>mkdir -p %{buildroot}/etc/rc.d/init.d/</script>
+                                    </installScriptlet>
+                                    <directoryIncluded>false</directoryIncluded>
+                                </configuration>
+                            </execution>
+                            <execution>
+                                <id>generate-splice-secure</id>
+                                <goals>
+                                    <goal>attached-rpm</goal>
+                                </goals>
+                                <configuration>
+                                    <name>splicemachine-${envClassifier}-hbase-secure-conf</name>
+                                    <targetOS>linux</targetOS>
+                                    <classifier>splicemachine</classifier>
+                                    <version>${project.version}.${parcel.patch}</version>
+                                    <group>Applications/Internet</group>
+                                    <license>Copyright © 2018 Splice Machine Inc. Licensed under the Apache License, Version 2.0.</license>
```

Comment:
Should this be copyright 2019?  There are a few other occurrences in this PR.
Most of this PR seems to be just duplication of files for the new platform, so I'm not reviewing all of the duplicated content.  Wondering if we can unify some of these files in the future so they can be shared between like platforms.